### DISCUSSION TITLE
Welcome to FabricForPowerBI Discussions!

### DISCUSSION THREAD

**Discussion by jasonhimmelstein** [jasonhimmelstein/FabricForPowerBI#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by christinalanham** [comment]
Hello all! My name is Christina, based out of Houston, TX and I am the sole PBI person in my company. I learned about PBI in 2020 and have been leveraging ever since to bring together data from disparate systems to allow the company to make more informed decisions. 

I have a long way to go in learning PBI and now Fabric, but very much enjoying the journey!

**Comment by tjmcallister** [comment]
Hi Jason, Tim McAllister here.  Am I remembering correctly that we would have access to Fabric for 90 days with our FabNNN@... account?  I can't seem to get logged in any longer (fab316@fabricconf.onmicrosoft.com).  Thanks for your help and really appreciate all of the info received during the workshop.

**Reply by jasonhimmelstein** [reply]
The accounts were only available for the week. It was a bonus for us to be able to give out these accounts for the workshop. You can use the trial for most things, and an F2 SKU for the rest. To do Copilot you would need an F64 but nothing in this set of labs requires that.
